src/constants/index.ts

```diff
@@ -158,3 +158,5 @@ export const isDev = process.env.NETWORK === 'ganache';
 export const CDAPP_VERSION = version;
 
 export const STAKING_THRESHOLD = 10;
+
+export const MAX_DISPLAY_NAME = 20;
```

Comment:
Could we specify this is the max Colony display name? User display doesn't have a max, for instance.

Reply:
The variable is now named `MAX_COLONY_DISPLAY_NAME`